go/protocol/keybase_v1.go

```diff
@@ -217,6 +217,14 @@ type UserPlusKeys struct {
 	Uvv        UserVersionVector `codec:"uvv" json:"uvv"`
 }
 
+type MerkleTreeID int
```

Comment:
So this got added but protocol wasn't ever genned / checked in after so thats why this exists